tutorials/basic-topics/sorting/insertion-sort.md

```diff
@@ -11,4 +11,208 @@ keywords:
   - insertion sort
 ---
 
-<TutorialAuthors names="@TBC"/>
+<TutorialAuthors names="@Shivashish-rwt"/>
+
+## Overview
+
+Insertion sort is one of the sorting algorithms that sort the elements by placing an unsorted element in correct sorted order one at a time.
+
+First of all, let us understand how an sorted `array` (If you don't what an array is. Please refer to `tutorials/basic-topics/arrays.md`) looks like. Understanding an sorted array will give us the overview of the insertion sort.
```

Comment:
Readers won't know what `tutorials/basic-topics/arrays.md` is. Use hyperlink instead.